content/_data/roadmap/roadmap.yml

```diff
@@ -337,11 +337,11 @@ categories:
       link: https://github.com/jenkinsci/jenkinsfile-runner
       labels:
       - feature
-    - name: 'External Fingerprint Storage'
+    - name: 'JEP-226: External Fingerprint Storage'
       description: >
         Extend Jenkins to support storing artifact usage history in external databases.
```

Comment:
Would it make sense to also mention the reference implementation here?

Reply:
Added :)